truffleruby/src/main/java/org/truffleruby/language/loader/RequireNode.java

```diff
@@ -161,6 +162,14 @@ protected boolean require(VirtualFrame frame, String feature,
 
                 addToLoadedFeatures(frame, pathString);
 
+                if(feature.endsWith("rspec/support/ruby_features")) {
```

Comment:
We should also match if somebody requires `"rspec/support/ruby_features"` or `"rspec/support/ruby_features.rb"`.